_includes/modal.html

```diff
@@ -27,9 +27,9 @@ <h6 class="pattern-type">{{pattern.type | upcase }}</h6>
       {%else%}
       <img
         class="image-class"
-        data-snapshot-light="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/action-assets/design-assets/{{pattern.patternId}}-light.png"
-        data-snapshot-dark="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/action-assets/design-assets/{{pattern.patternId}}-dark.png"
-        src="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/action-assets/design-assets/{{pattern.patternId}}-dark.png"
+        data-snapshot-light="https://raw.githubusercontent.com/meshery/meshery.io/master/assets/images/logos/service-mesh-pattern.svg"
+        data-snapshot-dark="https://raw.githubusercontent.com/meshery/meshery.io/master/assets/images/logos/service-mesh-pattern.svg"
+        src="https://raw.githubusercontent.com/meshery/meshery.io/master/assets/images/logos/service-mesh-pattern.svg"
```

Comment:
```suggestion
     data-snapshot-light="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/action-assets/design-assets/{{pattern.patternId}}-light.png"
        data-snapshot-dark="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/action-assets/design-assets/{{pattern.patternId}}-dark.png"
        src="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/action-assets/design-assets/{{pattern.patternId}}-dark.png"
```

Reply:
Let's revert these changes, as we just need to change the fallback svg not the dark / light theme svgs.